WORKFLOWS.md

```diff
@@ -0,0 +1,25 @@
+# Reusable Workflows
+
+In addition to GitHub community health templates, this repository defines reusable workflows for the organization.
+
+## Continuous Integration
+
+To use the CI workflow:
+
+```yaml
+# In a package repository, in the file .github/workflows/continuous-integration.yml:
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - '[0-9]+.[0-9]+.x'
+      - 'refs/pull/*'
+      - 'renovate/*'
```

Comment:
@Ocramius suggested just removing these entirely and having
```yml
on:
  pull_request:
  push:
    branches:
    tags:
```

Reply:
I can make that change. The reason we had the original set was to prevent runs on arbitrary branches unless they were used for pull requests, but since that never actually happens in practice, we could remove that stipulation.

Reply:
Updated!

Reply:
Suggestion: drop branch restrictions overall - no point in having them.